spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/SpanReporter.java

```diff
@@ -14,23 +14,18 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.sleuth.event;
-
-import org.springframework.cloud.sleuth.Span;
+package org.springframework.cloud.sleuth;
 
 /**
- * <b>cs</b> - Client Sent. The client has made a request (a client can be e.g.
- * {@link org.springframework.web.client.RestTemplate}. This annotation depicts
- * the start of the span.
+ * Contract for reporting Sleuth spans for collection. For example to Zipkin.
+ *
+ * @author Marcin Grzejszczak
  *
- * @author Dave Syer
  * @since 1.0.0
  */
-@SuppressWarnings("serial")
-public class ClientSentEvent extends SpanContainingEvent {
-
-	public ClientSentEvent(Object source, Span span) {
-		super(source, span);
-	}
-
-}
\ No newline at end of file
+public interface SpanReporter {
+	/**
+	 * Reports a Span for collection
```

Comment:
Reports a completed span out of band, usually out of process. This is typically to a trace depot (ex zipkin) or a log file.